hangman.py

```diff
@@ -113,21 +124,68 @@ def hangman(secret_word):
     
     Follows the other limitations detailed in the problem write-up.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    print('''
+    __  __                                           ___ ____ 
+   / / / /___ _____  ____ _____ ___  ____ _____     <  // __ \\
+  / /_/ / __ `/ __ \/ __ `/ __ `__ \/ __ `/ __ \    / // / / /
+ / __  / /_/ / / / / /_/ / / / / / / /_/ / / / /   / // /_/ / 
+/_/ /_/\__,_/_/ /_/\__, /_/ /_/ /_/\__,_/_/ /_/   /_(_)____/  
+                  /____/                                      
+''')
+    # print(secret_word)
+    print(f'I am thinking of a word that is {len(set(secret_word))} letters long.')
+    guesses = 6
+    letters_guessed = []
+    vowels = ['a', 'e', 'i', 'o', 'u']
+    warnings = 3
+    while 1:
+        if is_word_guessed(secret_word, letters_guessed):
+            total_score = guesses * len(set(secret_word))
+            print(f'\033[7mCongratulations! You won, the secret word was - {secret_word}\033[0m')
```

Comment:
This peace of code means style of string. I decided that it will improve user experience bcs the user can see the result of the game better and it highlights that user entered wrong value for letter. In my IDE it look like this:
![console](https://user-images.githubusercontent.com/56088929/100385027-3a4fcc80-302a-11eb-8c5d-e60e90e548e1.png)
I'm sorry that I forget to ask you about this feature. Could I left it?